src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java

```diff
@@ -6,6 +6,7 @@
 import uk.gov.hmcts.reform.bsp.common.model.shared.in.OcrDataField;
 import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant;
 import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.ChildInfo;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.ChildrenList;
```

Comment:
One last (optional) suggestion. Should this class be called ChildrenInfo, instead of ChildrenList?

Reply:
ok

Reply:
done